EvmYul/EVM/PrecompiledContracts.lean

```diff
@@ -442,3 +443,21 @@ private example :
     =
   (ByteArray.ofBlob "08c9bcf367e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d282e6ad7f520e511f6c3e2b8c68059b9442be0454267ce079217e1319cde05b").toOption
 := by native_decide
+
+def Ξ_PointEval
+  (σ : AccountMap)
+  (g : UInt256)
+  (A : Substate)
+  (I : ExecutionEnv)
+    :
+  (AccountMap × UInt256 × Substate × ByteArray)
+:=
+  let d := I.inputData
+  let gᵣ : UInt256 := 50000
```

Comment:
Wouldn't it make sense to define \xi.g_r := 50000 as a constant definition rather than putting it in the body?

Reply:
I don't think so. In some cases like `Ξ_SHA256` `Ξ_RIP160`, `gᵣ` depends on the size of the input data. In the case of `Ξ_EXPMOD` it depends on the lenght of `M` and `B`.